drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -82,6 +83,311 @@
 #define CEC_CLOCK_FREQ 40000
 #define VC4_HSM_CLOCK 163682864
 
+#define HDMI_CODEC_CHMAP_IDX_UNKNOWN  -1
+
+/*
+ * CEA speaker placement for HDMI 1.4:
+ *
+ *  FL  FLC   FC   FRC   FR   FRW
```

Comment:
FRW?

Reply:
From: https://elixir.bootlin.com/linux/v5.4.51/source/sound/soc/codecs/hdmi-codec.c#L31
I've checked CEA-861 where the other channels come from and it's not there. Also doesn't appear in enum below.